chains/evm/contracts/pools/TokenPool.sol

```diff
@@ -59,10 +59,8 @@ abstract contract TokenPool is IPoolV1, Ownable2StepMsgSender {
   error OverflowDetected(uint8 remoteDecimals, uint8 localDecimals, uint256 remoteAmount);
   error InvalidDecimalArgs(uint8 expected, uint8 actual);
 
-  event Locked(address indexed sender, uint256 amount);
-  event Burned(address indexed sender, uint256 amount);
-  event Released(address indexed sender, address indexed recipient, uint256 amount);
-  event Minted(address indexed sender, address indexed recipient, uint256 amount);
+  event LockedOrBurned(address indexed sender, uint256 amount);
+  event ReleasedOrMinted(address indexed sender, address indexed recipient, uint256 amount);
```

Comment:
Merged the events

Reply:
Don't you want to also add `token` here, in line with that added to `Out|InbountRateLimitConsumed`? And maybe to any other event which touches tokens or wanna-be token-specific codepaths?